Fix long contract name issue

YegorSan opened the pull request:
This PR is related to #2527 issue.

## Motivation

View for long contract names was broken.

## Changelog



### Bug Fixes
Fixed long contract name issue. 
![Screenshot (3)](https://user-images.githubusercontent.com/10085945/66295717-a803db00-e8f4-11e9-937d-a64f718e78b8.png)



Working now on Transaction Details page as bug appearing not only in internal transaction tile: 
![Screenshot (4)](https://user-images.githubusercontent.com/10085945/66295861-f44f1b00-e8f4-11e9-8f21-0938db828b9c.png)



## Checklist for your PR

<!--
  Ideally a PR has all of the checkmarks set.

  If something in this list is irrelevant to your PR, you should still set this
  checkmark indicating that you are sure it is dealt with (be that by irrelevance).

  If you don't set a checkmark (e. g. don't add a test for new functionality),
  you must be able to justify that.
-->

  - [ ] I added an entry to `CHANGELOG.md` with this PR
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from silently reappearing again.
  - [ ] I checked whether I should update the docs and did so if necessary
  - [ ] If I added/changed/removed ENV var, I should update the list of env vars in https://github.com/poanetwork/blockscout/blob/master/docs/env-variables.md to reflect changes in the table here https://poanetwork.github.io/blockscout/#/env-variables?id=blockscout-env-variables. I've set `master` in the `Version` column.
  - [ ] If I add new indices into DB, I checked, that they don't redundant with PGHero or other tools

vbaranov commented on apps/block_scout_web/assets/css/_layout.scss:
```diff
@@ -11,4 +11,11 @@
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+{
```
@YegorSan I believe, this is a typo. Please check tests after you make an update.

YegorSan replied:
@vbaranov Ok Victor, I'll re check tests and push an update.

YegorSan commented:
@vbaranov Hi Victor. Sorry for such a mistake. I've fixed braces typo and build test is passing now. Will push a small fix for long contract name in transaction tile in mobile view in 5 minutes.

vbaranov commented:
@YegorSan please attach screenshots of how the screens will look with this update

YegorSan commented:
@vbaranov Hello Victor. Unfortunately, I can not reproduce this issue locally (can't find contract with such a long name), so I can not provide the screenshot with the performed changes. I've worked on that issue in chrome inspector on stg app, and then performed all needed changes locally.

igorbarinov commented:
> @YegorSan
> 
> ![Screenshot 2019-10-08 at 22 42 02](https://user-images.githubusercontent.com/4341812/66400749-04fab080-ea1d-11e9-8e77-adcf77fe1306.png)

how it's implemented in etherscan @vbaranov

YegorSan commented:
@vbaranov Hi Victor. I've fixed issue with overflow: ellipsis, as it not worked as it expected with flexbox (not showing three dots if truncated). Now I've added display: flex to the parent span element and looks like it works fine now. I want to add that I've deleted unnecessary truncation for contract name with the standard length:

![Screenshot (7)](https://user-images.githubusercontent.com/10085945/66461384-9a2f9080-ea81-11e9-94b9-0877ae3f2f52.png)

For testing purposes I've hardcoded contract name value, and it will truncate string if the length is larger than standard legth:

![Screenshot (8)](https://user-images.githubusercontent.com/10085945/66461571-ff838180-ea81-11e9-8ee0-a8f02fc1d4ff.png)
 
Could you please review? Thank's in advance.

vbaranov commented:
>how it's implemented in etherscan @vbaranov

@igorbarinov Etherscan doesn't show Contract name on the tx tile at all

YegorSan commented:
@vbaranov Checking now.

igorbarinov commented:
> > how it's implemented in etherscan @vbaranov
> 
> @igorbarinov Etherscan doesn't show Contract name on the tx tile at all

IMHO it should be with `..` like with addresses 

`Longcontract` should be `Longco..ct`
